cicd/gitops/preventing-auto-reboot-during-argo-cd-sync-with-machine-configurations.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="preventing-auto-reboot-during-argo-cd-sync-with-machine-configurations"]
+
+= Preventing auto-reboot during Argo CD sync with Machine Configurations
+include::_attributes/common-attributes.adoc[]
+:context: preventing-auto-reboot-during-argo-cd-sync-with-machine-configurations
+
+In the Red Hat OpenShift Container Platform, nodes are updated automatically through the Red Hat OpenShift Machine Config Operator (MCO). A Machine Config Operator (MCO) is a custom resource that is used by the cluster to manage the complete life cycle of its nodes. When an MCO resource is created or updated in a cluster, the MCO picks up the update, performs the necessary changes to the selected nodes, and restarts the nodes gracefully by cordoning, draining, and rebooting those nodes. It handles everything from the kernel to the kubelet.
```

Comment:
I assume you don't do one sentence per line. https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line